packages/discord.js/src/structures/SoundboardSound.js

```diff
@@ -0,0 +1,198 @@
+'use strict';
+
+const { DiscordSnowflake } = require('@sapphire/snowflake');
+const { Base } = require('./Base.js');
+const { DiscordjsError, ErrorCodes } = require('../errors/index.js');
+
+/**
+ * Represents a soundboard sound.
+ * @extends {Base}
+ */
+class SoundboardSound extends Base {
+  constructor(client, data) {
+    super(client);
+
+    /**
+     * The id of the soundboard sound
+     * @type {Snowflake|string}
+     */
+    this.soundId = data.sound_id;
+
+    this._patch(data);
+  }
+
+  _patch(data) {
+    if ('available' in data) {
+      /**
+       * Whether this soundboard sound is available
+       * @type {?boolean}
+       */
+      this.available = data.available;
+    } else {
+      this.available ??= null;
+    }
+
+    if ('name' in data) {
+      /**
+       * The name of the soundboard sound
+       * @type {?string}
+       */
+      this.name = data.name;
+    } else {
+      this.name ??= null;
+    }
+
+    if ('volume' in data) {
+      /**
+       * The volume of the soundboard sound, from 0 to 1
+       * @type {?number}
+       */
+      this.volume = data.volume;
+    } else {
+      this.volume ??= null;
+    }
+
+    if ('emoji_id' in data) {
+      /**
+       * The emoji id of the soundboard sound
+       * @type {?Snowflake}
+       */
+      this.emojiId = data.emoji_id;
+    } else {
+      this.emojiId ??= null;
+    }
+
+    if ('emoji_name' in data) {
+      /**
+       * The emoji name of the soundboard sound
+       * @type {?string}
+       */
+      this.emojiName = data.emoji_name;
+    } else {
+      this.emojiName ??= null;
```

Comment:
We should probably turn this into just an `emoji` prop. Perhaps similar to the one in `WelcomeChannel`